src/NHC2.ts

```diff
@@ -33,7 +34,7 @@ export class NHC2 {
           flatMap(event => event.Params),
           map(params => params.Devices),
         )
-        .subscribe(devices => resolve(devices.filter(device => device.Type === 'action')));
+        .subscribe(devices => resolve(devices.filter(device => device.Online === 'True' && device.Type !== 'gatewayfw')));
```

Comment:
Out of curiosity, why do we need to filter on `Online` devices? What are examples of not-`Online` Niko devices?

Reply:
The 'energycontrol' module is something that is a 'generic' device and is offline, so it should not be visible I think.